api/api.go

```diff
@@ -683,13 +687,21 @@ func (api *Server) GetLogs(
 		if req.Count > 1000 {
 			return nil, status.Error(codes.InvalidArgument, "maximum query range is 1000 blocks")
 		}
-		startBlock = req.FromBlock
-		count = req.Count
+		startBlock := req.FromBlock
+		count := req.Count
+		logs, err = api.getLogsInRange(logfilter.NewLogFilter(in.GetFilter(), nil, nil), startBlock, startBlock+count-1)
+	case in.GetByLongRange() != nil:
```

Comment:
if this is supported, why should we keep GetByRange which has a limit of count?

Reply:
We could unify those two getLogsRequest, pls confirm this! https://github.com/iotexproject/iotex-proto/pull/102